docs/src/test-runners-csharp.md

```diff
@@ -299,7 +299,7 @@ dotnet test
 You can also choose specifically which tests to run, using the [filtering capabilities](https://docs.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=mstest):
 
 ```bash
-dotnet test --filter "Name~ShouldAdd"
+dotnet test --filter "Name~Slogan"
```

Comment:
Running this command against the examples result that no tests are executed.
I thought it would be better to filter a test from the examples while still having one test executed, if this is unwanted I can revert this change.